Sources/ShopifyAcceleratedCheckouts/Wallets/ApplePay/ApplePayAuthorizationDelegate/ApplePayAuthorizationDelegate+Errors.swift

```diff
@@ -68,5 +68,56 @@ extension ApplePayAuthorizationDelegate {
                 withLocalizedDescription: "Invalid shipping address"
             )
         }
+
+        static func shippingCountryNotSupported(supportedCountries: Set<String>) -> Error {
+            let message = formatCountryListMessage(supportedCountries: supportedCountries)
+            return PKPaymentRequest.paymentShippingAddressInvalidError(
+                withKey: "country",
+                localizedDescription: message
+            )
+        }
+
+        /// Apple Pay guideline: Keep error messages at 85 characters or fewer to avoid truncation in the UI
+        private static let applePayErrorMessageMaxLength = 85
```

Comment:
The official guideline is to keep error messages at 128 or less, but I've tested it in a iPhone 16 Pro Max and even there the limit is 85 to avoid truncation. In phones with smaller screens is even less. That's why I haven't used 128 as the limit. We could even consider lowering the limit to improve the UX for users with smaller screens.

Reply:
Max devices are pretty big. Do the 85 chars fit on the small version too? i.e. 16 Pro?

Reply:
No, on small versions won't fit if there are multiple shipping countries. I think only 2 or 3 fit on the screen.